script/openqa-bootstrap

```diff
@@ -105,3 +110,10 @@ EOF
 
 # start worker
 systemctl enable --now openqa-worker@1.service
+
+# clone jobs if jobId(s) are given
+if [ $# -ne 0 ]; then
+    for arg; do
+	sudo /var/lib/openqa/script/clone_job.pl -from openqa.opensuse.org "$arg"
```

Comment:
This should be at best just `openqa-clone-job "$@"`. I would not call clone jobs multiple times here, *however* it is a neat idea. We could extend this actually in `openqa-clone-job` itself to iterate over each job-url or id.

I would simply forward *all* additional arguments so that we can also pass further job parameters to overwrite the settings on the cloned job. Example call:

```
openqa-bootstrap --from https://openqa.opensuse.org 12345 SCHEDULE=tests/boot/boot_to_desktop,tests/x11/kontact
```

Also, please no "sudo". There should be no root-permissions required. The authentication is handled by the api key+secret, not unix permissions.